client/src/pages/SubmitPage/ImageElement/ImageElement.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { Box, Paper, ImageList, ImageListItem, IconButton } from '@material-ui/core';
+import DeleteOutlineOutlinedIcon from '@material-ui/icons/DeleteOutlineOutlined';
+
+import useStyles from './useStyles';
+
+interface ImageListProps {
+  file: File;
+  filename: string;
+  status: string;
+  storageRef: string;
+  downloadUrl: string;
+  description: string;
+}
+
+interface ArrayImageList {
+  imageList: ImageListProps[];
+  handleDeleteImage: (index: number) => void;
+}
+
+function ImageElement({ imageList, handleDeleteImage }: ArrayImageList): JSX.Element {
+  const classes = useStyles();
+
+  return (
+    <Box my={2}>
+      <Paper>
+        <ImageList rowHeight={160} cols={4}>
+          {imageList.map((image, index) => (
+            <ImageListItem key={image.storageRef} style={{ maxHeight: '17.5em' }}>
```

Comment:
Avoid using indexes as keys, try to use something unique. When no unique keys are available, you can consider using a package like uuid :)

Reply:
I did not use index as keys here. I used the generated image urls as key 
```
<ImageListItem key={image.storageRef} style={{ maxHeight: '17.5em' }}>
```
Where the key is image.storageRef

Reply:
My apologies, I was thinking about another PR I guess 😄

Reply:
😄 I understand.